pkg/util/log/BUILD.bazel

```diff
@@ -206,7 +206,7 @@ genrule(
 
 gomock(
     name = "mock_logsink",
-    out = "mock_generated.go",
+    ougt = "mock_generated.go",
```

Comment:
?

Reply:
typo, fixed, or, at least, I thought I did, github doesn't seem to see it.